atlp-rwanda/e-comm-team-techsmith-fn
# #185357389 Redux toolkit configuration

## What does this PR do?
This PR aims at configuring redux toolkit in our project

## Description of Task to be completed?
This task was about setting up redux toolkit which is a library that provides utilities and abstractions to simplify the development of Redux applications. It is an official package from the Redux team and is built on top of Redux. Redux Toolkit aims to streamline and standardize common Redux use cases, reduce boilerplate code, and improve developer experience.
## How should this be manually tested?
- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ch-redux-toolkit-185357389`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- For easy check out make sure you have the redux dev tools installed if not configure them from this link [here](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd)
## What are the relevant pivotal tracker/Trello stories?
[#185357389](https://www.pivotaltracker.com/story/show/185357389)
## Screenshots (if appropriate)
![Screen Shot 2015-10-01 at 1 25 24 AM](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/e5ab4320-3458-47f5-a6b5-5a3a92a4e045)
